src/draw.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python3
```

Comment:
Since it would be hard to write tests for this, and this code is just a tool for helping with data exploration/debugging I didn't really want to write tests for it. If you think it needs some we can talk tomorrow (Wednesday) before lecture.

Reply:
ah! If its for exploration and debugging dont worry about testing. Maybe just note that in the doc string :)